specs/option_spec.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require 'rspec'
+require_relative '../option'
+
+RSpec.describe Option do
+  let(:option1) { described_class.new('Option 1') }
+
+  describe '#initialize' do
+    it 'creates an instance of Option class' do
+      expect(option1).to be_an(Option)
+    end
+
+    it 'assigns text correctly during initialization' do
+      expect(option1.text).to eq('Option 1')
+    end
+  end
+
+  describe 'attr_accessor :text' do
+    it 'allows reading the text attribute' do
+      expect(option1.text).to eq('Option 1')
+    end
+
+    it 'allows modifying the text attribute' do
+      option1.text = 'New Option 1'
+      expect(option1.text).to eq('New Option 1')
+    end
+  end
```

Comment:
Actually, these are 2 methods:
1. `.text` -> attr_reader
2. `.text=` -> attr_writer

```suggestion
  describe '#text' do
    it 'returns text value' do
      expect(option1.text).to eq('Option 1')
    end
  end

  describe '#text=' do
    it 'updates text' do
      expect(option1.text).to eq('Option 1')

      option1.text = 'New Option 1'

      expect(option1.text).to eq('New Option 1')
    end
  end
```

Reply:
For `text=` May be we can do something like 

```ruby
expect { option1.text = 'New Option 1' }.to change { option1.text }.from('Option 1').to('New Option 1')
```